nms/app/packages/magmalte/app/components/lte/LteContext.js

```diff
@@ -106,13 +107,14 @@ export function GatewayContextProvider(props: Props) {
     <GatewayContext.Provider
       value={{
         state: lteGateways,
-        setState: (key, value?) => {
+        setState: (key, value?, refreshingState?: {[string]: lte_gateway}) => {
           return SetGatewayState({
             lteGateways,
             setLteGateways,
             networkId,
-            key,
+            key: key,
             value,
+            refreshingState: refreshingState,
```

Comment:
Instead of calling this refreshing state, can you call this newState and you can just use the shorthand dictionary notation and call setGatewayState with (lteGateways, setLteGateways, networkId, key, value, newState)